CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ first, also the PHP version required is 7.1, see [#426].
 - Listen for Issue/Note GitLab events (@glensc, #337, #448)
 - xmlrpc: verify token first to avoid incrementing failed logins (@glensc, #463)
 - Do issue duplication and close in same step (@glensc, #127)
+- Fix open issues report: set default number of days to exclude
```

Comment:
please add PR Id (and your github handle if you wish).

you should git fixup and force push again

Reply:
OK, done.

Reply:
you still have 3 commits:
- https://github.com/eventum/eventum/pull/458/commits

you still have WIP in title (wip indicates your development is not finished, must not be merged)

Reply:
Thanks! git is a kind of black magic. 
Squashed three commits into one, removed WIP in the title.